scripts/deployment/_push-assets-to-cdn

```diff
@@ -118,10 +135,11 @@ def get_mimetype_temp_dir(mimetype):
   mimetype_folder = encode_dirname_with_slash(mimetype)
   return f'{temp_dir}/{mimetype_folder}'
 
-mimetypes_to_process = []
+
+mimetypes_to_process = set()
 for f in static_files_to_copy:
   mimetype = mime_type_for(f)
-  mimetypes_to_process.append(mimetype)
+  mimetypes_to_process.add(mimetype)
```

Comment:
LOL

Reply:
The change to `set()` was the important change